### PR TITLE
Migrate tracking DQM to phase1 and enable it

### PR DESCRIPTION
**Pull request by makortel**
This PR adds the phase1 iterations to tracking DQM configuration for seeding/building plots.

Tested in CMSSW_8_1_X_2016-04-24-0000, the only expected change is an addition of histograms in `Tracking/TrackParameters` folder in 2017 workflows.

@rovere @VinInn @mtosi

### CODE REVIEW

File: DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py [hunk 1]
```diff
@@ -110,14 +135,53 @@
...
+    "highPtTripletStep",
+    "lowPtQuadStep",
+    "lowPtTripletStep",
+    "detachedQuadStep",
+    "mixedTripletStep",
+    "pixelPairStep",
+    "tobTecStep",
+    "muonSeededStepInOut",
+    "muonSeededStepOutIn",
+]
```
Comment (makortel) [inline DQM/TrackingMonitorSource/python/IterTrackingModules4seedMonitoring_cfi.py]: This is now the third place where the same information about iterations is copy-pasted (two others being https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoTracker/IterativeTracking/python/iterativeTk_cff.py and https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Validation/RecoTrack/python/TrackValidation_cff.py). To ease maintenance I'd very much like to specify the information only once (therefore that must be under `RecoTracker/IterativeTracking`). I have already an idea for that and will prototype that.

**Comment by makortel** [comment]
Could DQM (@degio @dmitrijus @vanbesien) please review and sign? (And could somebody launch the tests?) Thanks.

**Comment by deguio** [comment]
please test

**Comment by makortel** [comment]
ping

**Comment by makortel** [comment]
ping

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**